src/server/src/modules/mail.ts

```diff
@@ -0,0 +1,32 @@
+import nodemailer from 'nodemailer';
+
+const rRatchEmailAddress = 'rmatch.jmsv@gmail.com'
+
+const transporter = nodemailer.createTransport(
+    `smtps://rmatch.jmsv%40gmail.com:ucrcs178winter2021@smtp.gmail.com`
+);
+
+/**
+ * @description Send an email to the specified email address.
+ * @param {string} email - Receiver email address
+ * @param {string} subject - Email subject
+ * @param {string} text - Email text
+ * @returns Promise
+ */
+export const sendEmail = async (
+    email: string,
+    subject: string,
+    text: string
+) => {
+    const mailOptions = {
+        from: rRatchEmailAddress,
+        to: email,
+        subject: subject,
+        text: text
+    };
+
+    transporter.sendMail(mailOptions, (error, info) => {
+        if (error) return console.log(error);
+        console.log('Message sent: ' + info.response);
```

Comment:
Same here, import the logger and use it to print your message like `logger.info('Message sent....')`

Reply:
Done.